### PR TITLE
RDK-35331: RDKShell - Key Repeat Updates

### PR DESCRIPTION
**Pull request by mateusz-hobgarski-red**

### CODE REVIEW

File: RDKShell/RDKShell.cpp [hunk 1]
```diff
@@ -126,6 +126,7 @@ const string WPEFramework::Plugin::RDKShell::RDKSHELL_METHOD_GET_GRAPHICS_FRAME_
 const string WPEFramework::Plugin::RDKShell::RDKSHELL_METHOD_SET_GRAPHICS_FRAME_RATE = "setGraphicsFrameRate";
 const string WPEFramework::Plugin::RDKShell::RDKSHELL_METHOD_SET_AV_BLOCKED = "setAVBlocked";
 const string WPEFramework::Plugin::RDKShell::RDKSHELL_METHOD_GET_AV_BLOCKED_APPS = "getBlockedAVApplications";
+const string WPEFramework::Plugin::RDKShell::RDKSHELL_METHOD_KEY_REPEAT_CONFIG = "keyRepeatConfig";
```
Comment (anand-ky) [inline RDKShell/RDKShell.cpp:129]: Pls update Documentation for the new APIs by updating RDKShell.json.

**Review by anand-ky — CHANGES_REQUESTED** [review]

File: RDKShell/RDKShell.cpp (hunk 1 above)
Comment (mateusz-hobgarski-red) [reply RDKShell/RDKShell.cpp:129]: updated

**Review by anand-ky — APPROVED** [review]

**Comment by anand-ky** [comment]
@mateusz-hobgarski-red - Pls resolve conflicts since PR 2809 got merged.

**Comment by anand-ky** [comment]
Closing older PRs. Pls bring changes to latest sprint.

**Pull request closed without merging**